_monitoring-your-cluster/pa/index.md

```diff
@@ -8,57 +8,106 @@ redirect_from:
+The Performance Analyzer plugin is included in the installation for [Docker]({{site.url}}{{site.baseurl}}/opensearch/install/docker/) and [tarball]({{site.url}}{{site.baseurl}}/opensearch/install/tar/), but you can also install the plugin manually. 
 
-To start the performance analyzer root cause analysis (RCA) agent on a tarball installation, run the following command:
+To install the Performance Analyzer plugin manually, download the plugin from [Maven](https://search.maven.org/search?q=org.opensearch.plugin) and install it using the standard [plugins install]({{site.url}}{{site.baseurl}}/opensearch/install/plugins/) process. Performance Analyzer runs on each node in a cluster.
+
+To start the Performance Analyzer root cause analysis (RCA) agent on a tarball installation, run the following command:
       
 ````bash
 OPENSEARCH_HOME=~/opensearch-2.2.1 OPENSEARCH_JAVA_HOME=~/opensearch-2.2.1/jdk OPENSEARCH_PATH_CONF=~/opensearch-2.2.1/bin ./performance-analyzer-agent-cli
 ````
 
-The following command enables the performance analyzer plugin and performance analyzer RCA agent:
+The following command enables the Performance Analyzer plugin and Performance Analyzer RCA agent:
 
 ````bash
 curl -XPOST localhost:9200/_plugins/_performanceanalyzer/cluster/config -H 'Content-Type: application/json' -d '{"enabled": true}'
 ````
 
-To shut down the performance analyzer RCA agent, run the following command:
+## Disable Performance Analyzer
 
-````bash
-kill $(ps aux | grep -i 'PerformanceAnalyzerApp' | grep -v grep | awk '{print $2}')
-````
+If you prefer to save memory and run your local instance of OpenSearch with the Performance Analyzer plugin disabled, use the following steps.
 
-To disable the performance analyzer plugin, run the following command:
+1. Before disabling Performance Analyzer, stop any currently running RCA agent action using the following command:
 
-````bash
-curl -XPOST localhost:9200/_plugins/_performanceanalyzer/cluster/config -H 'Content-Type: application/json' -d '{"enabled": false}'
-````
+  ```bash
+  curl -XPOST localhost:9200/_plugins/_performanceanalyzer/rca/cluster/config -H 'Content-Type: application/json' -d '{"enabled": false}'
+  ```
 
-To uninstall the performance analyzer plugin, run the following command:
+2. Shut down the Performance Analyzer RCA agent by running the following command:
 
-````bash
-bin/opensearch-plugin remove opensearch-performance-analyzer
-````
+  ```bash
+  kill $(ps aux | grep -i 'PerformanceAnalyzerApp' | grep -v grep | awk '{print $2}')
+  ```
+
+2. Disable the Performance Analyzer plugin by running the following command:
```

Comment:
Numbering is incorrect; repeating 2. twice

Reply:
Fixed